src/main.js

```diff
@@ -72,4 +73,10 @@ if (!window.WebVRConfig.DEFER_INITIALIZATION) {
   window.InitializeWebVRPolyfill = function() {
     new WebVRPolyfill();
   }
+  // Call this if you want to use the shim without the rest of the polyfill.
+  // InitializeWebVRPolyfill() will install the shim automatically when needed,
+  // so this should rarely be used.
+  window.InitializeSpecShim = function() {
```

Comment:
Agree. When would this global be called? Also why do we have the two different names `InstallWebVRSpecShim` and `InitializeSpecShim` ?